lib/ansible/modules/network/aci/aci_anp.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+
+module: aci_anp
+short_description: Manage top level application network profile objects
+description:
+    -  Manage top level application network profile object, i.e. this does
+      not manage EPGs.
```

Comment:
Indentation error. Spacing is _very_ important.
And while we're at it, you can move that whole block to the start of the line.